src/libs/actions/IOU.js

```diff
@@ -2302,6 +2302,10 @@ function navigateToNextPage(iou, iouType, report, path = '') {
     Navigation.navigate(ROUTES.MONEY_REQUEST_PARTICIPANTS.getRoute(iouType));
 }
 
+function submitReport() {
+    // TODO: this will be done in the future, part of wave 6: drafts
```

Comment:
Can we link to the issue https://github.com/Expensify/App/issues/28763 instead of the todo